security/handler/security-handler-saml/src/main/java/org/codice/ddf/security/handler/saml/SAMLAssertionHandler.java

```diff
@@ -102,6 +115,44 @@ public HandlerResult getNormalizedToken(ServletRequest request, ServletResponse
             return handlerResult;
         }
 
+        String authHeader = ((HttpServletRequest) request)
+                .getHeader(SAML_HEADER_NAME);
+
+        // check for full SAML assertions coming in via Headers (backport as per DDF-1472)
```

Comment:
You don't need to say it is a back port and the ticket number should be part of the commit message.